index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="index.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>My Logo</header>
+    <h1>I’m Mariane Glad to see you!</h1>
+    <p>
+      I’m a software developer! I can help you build a product , feature or
+      website Look through some of my work and experience! If you like what you
+      see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <p>LET’S CONNECT</p>
+  </body>
```

Comment:
- [ ] [REQUIED] Please, add a section for the social media links as seen in the Figma Design